libs/java/auth_core/src/test/java/com/yahoo/athenz/auth/token/TokenTest.java

```diff
@@ -116,15 +116,15 @@ public void testTokenValidateNoExpiryTimestamp() throws CryptoException {
         long timestamp = System.currentTimeMillis() / 1000;
         PrincipalToken token = new PrincipalToken.Builder(svcVersion, svcDomain, svcName)
             .host(host).salt(salt).issueTime(timestamp).build();
-        token.expiryTime = 0;
+        token.expiryTime = 10;
```

Comment:
Setting expiryTime as 0 will make the condition to be always true so I changed it to 10.